playwright/components/draggable/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { Page } from "@playwright/test";
+import { DRAGGABLE_ITEM, DRAG_ICON } from "./locators";
+
+export const draggableItem = (page: Page, text: string) =>
```

Comment:
**question**: is there a better way we can do this, even just by adding some sort of tag, as this makes me genuinely scared